api/src/main/resources/application.properties

```diff
@@ -23,3 +23,5 @@ spring.datasource.max-idle=5
 spring.datasource.min-idle=1
 
 spring.jpa.hibernate.ddl-auto=none
+
+logging.level.org.pmiops.workbench=INFO
```

Comment:
Nit: newline

Reply:
So I think the default log level in all environments right now is INFO already.

In any case, I'd remove this and log the above at WARNING / ERROR level.

Reply:
So, weirdly enough, without this line we weren't getting logs on our local. So I'm wondering if it broke somehow and we don't get default log level any more?

Reply:
I'll double check just to be sure.

Reply:
NM, it does work without the line, new version up now.